src/pages/iou/request/step/IOURequestStepWaypoint.tsx

```diff
@@ -169,6 +172,24 @@ function IOURequestStepWaypoint({
         goBack();
     };
 
+    const onScroll = useCallback(() => {
+        if (!isSafari()) {
```

Comment:
Not 100% sure this meets our guidelines for cross-platform code, but it's a grey area